src/tiledb/cloud/soma/mapper.py

```diff
@@ -0,0 +1,436 @@
+import logging
+from typing import Any, Callable, Dict, Optional, Sequence, Tuple
+
+import anndata as ad
+import tiledbsoma
+
+from tiledb.cloud import dag
+from tiledb.cloud._common import functions
+
+_DEFAULT_RESOURCES = {"cpu": "8", "memory": "8Gi"}
+"""Default resource size; equivalent to a "large" UDF container."""
+
+
+def run_collection_mapper_workflow(
+    *,
+    # Input data:
+    soma_collection_uri: Optional[str] = None,
+    soma_experiment_uris: Optional[Sequence[str]] = None,
+    experiment_names: Optional[Sequence[str]] = None,
+    measurement_name: str,
+    X_layer_name: str,
```

Comment:
@JohnMoutafis this idiom is intentional: such parameters must be specified, and they must be specified by name.

Otherwise, positionality-only means people can lose count of positional arguments too easily.